package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "@types/node": "^12.6.9",
```

Comment:
I'm not sure what the purpose of this in `devDependencies` is...

we don't build anything with typescript and we're the only ones who would install the devDeps

either we

- move this to `dependencies` if it's really required to compile with our types (but we just need buffer could at least limit to `"@types/node": "*"?)
- don't add anything here because chances are if you're using a `Buffer` in your code you already needed to bring in node types somewhere and it's a fairly common thing for ts users to be familiar with rather than forcing these types to be installed for all our package consumers

I'm partial to option 2 myself. I'm comfortable as a ts user with packages not demanding random `@types/node` versions in exchange for installing myself

Reply:
Honestly, I probably agree with the second choice. I think I threw it into devDeps as I was working on it but must've just not thought about it while I was working on it